src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -134,4 +134,11 @@ function getIframeHosts(value) {
 :global(.htmlpopup-content) {
     padding: 7px;
 }
+:global(td) {
+    vertical-align: top;
+}
+
+:global(td.cell-left) {
+    padding-right: 10px;
+}
```

Comment:
Not sure it is a good idea to use `global` on a `td` element, does this changes for the whole application ? @pakb what do you think ?

Reply:
the alignment seems to work without the global. I thought we needed it to apply the style to the html popup, but it seems we can avoid using it, thus avoid side effects.

I'm removing the 'global'

and yes, the :global would have changed for the whole application